x/incentive/keeper/scenario_test.go

```diff
@@ -182,3 +182,44 @@ func TestBasicIncentivePrograms(t *testing.T) {
 		"bob pending rewards at time 300",
 	)
 }
+
+func TestZeroBonded(t *testing.T) {
+	const (
+		umee  = fixtures.UmeeDenom
+		uumee = leveragetypes.UTokenPrefix + fixtures.UmeeDenom
+	)
+
+	k := newTestKeeper(t)
+
+	// init a community fund with 1000 UMEE and 10 ATOM available for funding
+	k.initCommunityFund(
+		coin.New(umee, 1000_000000),
+	)
+
+	// create incentive program and fund from community
```

Comment:
let's use comments either to shortly explain the test case, notify something tricky (should be rarely) or explain something complex (also should be rarely). In most other cases, comments should be self explanatory. 
https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/

Reply:
simplified